spacepy/lib.py

```diff
@@ -91,8 +91,10 @@ def load_lib():
     Normally this will be in the directory where spacepy is installed,
     under libspacepy.
 
-    @return: the open library
-    @rtype: ctypes.CDLL or ctypes.WinDLL
+    Returns
+    =======
+    library : ctypes.DLL or ctypes.winDLL
+        the open library
```

Comment:
Does this render properly (linking to the types) with the "or"? At any rate this isn't true anymore, it's all ctypes.CDLL (note not DLL).

Reply:
I don't actually see a documentation section for lib, maybe its a keyword that sphinx isn't picking up on? I checked 0.7.0 docs and 0.6 docs and neither have documentation for lib

Reply:
It's not in our list of modules in index.rst, but it is linked if you click on the main top-level spacepy link in the list of modules. I think I prefer it that way, honestly.

Reply:
Resolved